src/App.vue

```diff
@@ -21,7 +22,16 @@ const searchDrugs = (drugSearch: string) => {
             })()
           )
         : (() => drug.openfda !== undefined && drug.openfda.brand_name !== undefined)()
-    ) as DrugProp[]
+    ) as DrugProp[];
```

Comment:
The above code is difficult to understand at a quick glance. Ternaries are great, I love them, but they should generally be limited to single line returns and not multi-line logic. Also, always document the code if you start writing something that might be complicated.

Reply:
[096002b](https://github.com/codingwithcn/OpenFDA-Vue/commit/096002b09e7f26f23adda863494bda79cdb10adc) fixed issue